allennlp/tests/common/from_params_test.py

```diff
@@ -132,6 +132,111 @@ def from_params(cls, params: Params, size: int) -> 'C':  # type: ignore
         assert c.name == "extra_c"
         assert c.size == 20
 
+    def test_extras_for_custom_classes(self):
```

Comment:
This file still has some mypy and pylint errors.

Reply:
Fixed.